internal/query/projection/settings_relational.go

```diff
@@ -0,0 +1,1768 @@
+package projection
+
+import (
+	"context"
+	"database/sql"
+	"encoding/json"
+	"net/url"
+
+	"github.com/zitadel/zitadel/backend/v3/domain"
+	"github.com/zitadel/zitadel/backend/v3/storage/database"
+	v3_sql "github.com/zitadel/zitadel/backend/v3/storage/database/dialect/sql"
+	db_json "github.com/zitadel/zitadel/backend/v3/storage/database/json"
+	"github.com/zitadel/zitadel/backend/v3/storage/database/repository"
+	"github.com/zitadel/zitadel/internal/eventstore"
+	"github.com/zitadel/zitadel/internal/eventstore/handler/v2"
+	"github.com/zitadel/zitadel/internal/repository/instance"
+	"github.com/zitadel/zitadel/internal/repository/org"
+	settings "github.com/zitadel/zitadel/internal/repository/organization_settings"
+	"github.com/zitadel/zitadel/internal/repository/policy"
+	"github.com/zitadel/zitadel/internal/zerrors"
+)
+
+const (
+	SettingsRelationalProjectionTable = "zitadel.settings"
+
+	SettingIDCol          = "id"
+	SettingInstanceIDCol  = "instance_id"
+	SettingsOrgIDCol      = "org_id"
+	SettingsTypeCol       = "type"
+	SettingsLabelStateCol = "label_state"
+	SettingsSettingsCol   = "settings"
+)
+
+type settingsRelationalProjection struct{}
+
+func newSettingsRelationalProjection(ctx context.Context, config handler.Config) *handler.Handler {
+	return handler.NewHandler(ctx, &config, new(settingsRelationalProjection))
+}
+
+func (*settingsRelationalProjection) Name() string {
+	return SettingsRelationalProjectionTable
+}
+
+func (s *settingsRelationalProjection) Reducers() []handler.AggregateReducer {
+	return []handler.AggregateReducer{
+		{
+			Aggregate: org.AggregateType,
+			EventReducers: []handler.EventReducer{
```

Comment:
Add `org.removed` event to then remove all settings associated with the organization.